ansible_collections/arista/avd/molecule/eos_designs_negative_unit_tests/inventory/host_vars/failure-l2vlan-parent-profile-does-not-exist.yml

```
@@ -0,0 +1,32 @@
+---
+type: l3leaf
+
+l2vlan_profiles:
+  - profile: child_profile
+    parent_profile: THIS-PROFILE-DOES-NOT-EXIST
+
+l3leaf:
+  defaults:
+    loopback_ipv4_pool: 10.10.0.0/24
+    vtep_loopback_ipv4_pool: 10.11.0.0/24
+    vtep_loopback: Loopback1
+    evpn_services_l2_only: false
+  nodes:
+    - name: failure-l2vlan-parent-profile-does-not-exist
+      id: 3
+      bgp_as: 65101
+      filter:
+        tags: [research]
+
+tenants:
+  - name: research
+    vlan_aware_bundle_number_base: 400
+    mac_vrf_vni_base: 10000
+    l2vlans:
+      - id: 110
+        name: research1
+        tags: ['research']
+        profile: child_profile
+
+expected_error_message: >-
+  Profile `THIS-PROFILE-DOES-NOT-EXIST` applied under L2VLAN Profile `child_profile` does not exist in `l2vlan_profiles`.
```

we normally use single quotes and not backtick.

```suggestion
  Profile 'THIS-PROFILE-DOES-NOT-EXIST' applied under L2VLAN Profile 'child_profile' does not exist in 'l2vlan_profiles'.
```